bin/compile

```diff
@@ -4,7 +4,7 @@ set -eu
 
 METABASE_DIR="$1/target/uberjar"
 METABASE_JAR="$METABASE_DIR/metabase.jar"
-METABASE_URL="https://downloads.metabase.com/enterprise/v1.45.1/metabase.jar"
+METABASE_URL="https://downloads.metabase.com/enterprise/latest/metabase.jar"
```

Comment:
@michaeldumalag: I'd like to specify the version of Metabase we're going to run instead of relying on latest.  Going to latest is not the same as going to 1.46.1.

```suggestion
METABASE_URL="https://downloads.metabase.com/enterprise/v1.46.1/metabase.jar"
```